platform_services/components/crud/getList.js

```diff
@@ -80,11 +80,11 @@ module.exports = (query, onComplete) => {
 				attributeValues[(":" + key_name)] = {
 					'S': query[key]
 				};
-			}   
+			}
 		});
     }
 
-    if (global.userId && !_.includes(global.config.admin_users, global.userId.toLowerCase())) {
+    if ((global.userId && !_.includes(global.config.admin_users, global.userId.toLowerCase())) || !getAllRecords) {
```

Comment:
small optimization based on how OR works [short circuit eval](https://en.wikipedia.org/wiki/Short-circuit_evaluation) - you want to move !getAllRecords to the front